### PR TITLE
Build basic instances page and modify routes

### PR DESCRIPTION
**Pull request by gauransh7**
Build basic instances page and modify routes

### CODE REVIEW

File: src/components/app.js [hunk 1]
```diff
@@ -19,13 +22,25 @@ class App extends Component {
...
-      <div styleName="main.app">
-        {/* <AppHeader appName="file_manager" mode="app" /> */}
+      <div styleName='main.app'>
+        {/* <AppHeader appName='file_manager' mode='app' /> */}
+        <AppHeader
+          mode='site'
+          userDropdown
+          appName='file_manager'
+          // onSidebarClick={() => ChangeSidebarVisibility(sidebarVisibility)}
+          // sideBarButton
```
Comment (ayu023ban) [inline src/components/app.js:30]: why it is commented out?

File: src/components/app.js [hunk 2]
```diff
@@ -35,7 +50,4 @@ class App extends Component {
   }
 }
 
-export default connect(
-  null,
-  null
-)(App)
+export default connect(null, null)(App)
```
Comment (ayu023ban) [inline src/components/app.js]: if it is null,null then why do you connect
just export as App

File: src/components/filemanager-card.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  render () {
+    const { filemanagername, contentSize, maxSpace } = this.props
+    let percentage = (contentSize * 100) / maxSpace
+    return (
+      <Card as={Link} to={`/file-manager/${filemanagername}`}>
+        <Card.Content>
+          <Image
+            floated='right'
+            size='mini'
+            src='https://img2.pngio.com/yellow-folder-transparent-png-clipart-free-download-ywd-yellow-folder-png-2400_1729.png'
```
Comment (ayu023ban) [inline src/components/filemanager-card.js:26]: why is the src of image hard coded?
it should be in the constants file. If it is not present then make a new file name constants

**Review by ayu023ban — CHANGES_REQUESTED** [review]

File: src/components/filemanager-card.js (hunk 3 above)
Comment (gauransh7) [reply src/components/filemanager-card.js:26]: it is a dummy image used just to check UI, We will be displaying unique images corresponding to the filemanager instance.
This image will not be used afterwords therefore i am not including it in constants for now.

File: src/components/app.js (hunk 2 above)
Comment (gauransh7) [reply src/components/app.js]: Yea .. it was like that from the previous commit. I have not done that intentionally ... Will change it !

**Pull request merged**